spec/controllers/entries_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe EntriesController, type: :controller do
+  describe 'GET #index' do
+    it "renders the index template" do
+      get :index, params: {list_id: 1}
+      expect(response).to render_template("index")
+    end
+  end
+  
+  describe 'GET #show' do
+    it "renders the show template" do
+      get :show, params: {list_id: 1, id: 1}
+      expect(response).to render_template('show')
+    end
+
+    it "returns a list with corresponding list_id" do
+      user = User.create!(id: 1, username: "Squid", email: "s@s.com", first_name: "Sid", last_name: "Monty")
+      list = List.create!(id: 2, title: "heyo", user: user)
+      get :show, params: {list_id: list.id, id: 1}
+      expect(assigns(:list)).to eq(list)
+    end
+
+    it "does not return a list with a different list_id" do
+      user = User.create!(id: 1, username: "Squid", email: "s@s.com", first_name: "Sid", last_name: "Monty")
+      list = List.create!(id: 2, title: "heyo", user: user)
+      get :show, params: {list_id: 1, id: 1}
+      expect(assigns(:list)).not_to eq(list)
```

Comment:
Not only do we expect that `assigns(:list)` to not equal `list` but because of how `List.find_by_id` works, we would also expect assigns(:list) to be nil.  So that's worth testing too.

To make that clearer you could write something like:
```
nonexistent_list = List.find_by_id(1)
expect(nonexistent_list).to eq nil

get :show, params: {list_id: 1, id: 1}

expect(assigns(:list)).to eq nil
expect(assigns(:list)).not_to eq list
```

This has a couple of benefits:
We ensure that list we're attempting to find in the route doesn't exist with the first expectation.  This has the added benefit of ensuring that if earlier in the test we (in some other PR) actually created List with an id of 1, we aren't giving a false positive.